geode-docs/README.md

```diff
@@ -1,93 +1,53 @@
-# Project Geode End-User Documentation
+# Apache Geode End-User Documentation
 
-The latest check-ins to `project-geode/docs` are automatically built and published to <http://geode-docs.cfapps.io>.
+Apache Geode provides the full source for end-user documentation in markdown format. The latest check-ins to `incubator-geode/geode-docs` are regularly built and published to http://geode.incubator.apache.org/docs/. Users can build the markdown into an HTML user guide using [Bookbinder](https://github.com/pivotal-cf/bookbinder) and the instructions below.
 
-Project Geode provides the full source for end-user documentation in markdown format. The markdown can be built into HTML using [Bookbinder](https://github.com/pivotal-cf/bookbinder).
+Bookbinder is a Ruby gem that binds  a unified documentation web application from markdown, html, and/or DITA source material. The source material for bookbinder must be stored either in local directories or in GitHub repositories. Bookbinder runs [middleman](http://middlemanapp.com/) to produce a Rackup app that can be deployed locally or as a Web application.
 
-Bookbinder is a gem that binds together a unified documentation web application from markdown, html, and/or DITA source material. The source material for bookbinder must be stored either in local directories or in GitHub repositories. Bookbinder runs [middleman](http://middlemanapp.com/) to produce a Rackup app that can be deployed locally or as a Web application.
+This document contains instructions for building and viewing the Geode documentation locally.
 
-This document contains instructions for building the local Geode documentation and publishing it to Cloud Foundry or Pivotal Web Services. It contains the sections:
+- [Prerequisites](#prerequisites)
+- [Bookbinder Usage](#bookbinder-usage)
+- [Building the Documentation](#building-the-documentation)
 
-- [Bookbinder Usage](#usage)
-- [Prerequisites](#prereq)
-- [Building the Documentation](#building)
-- [Publishing the Documentation](#publishing)
-- [Getting More Information](#moreinfo)
+## Prerequisites
 
-[]()
+Bookbinder requires Ruby version 2.0.0-p195 or higher.
 
-## Bookbinder Usage
+Follow the instructions below to install Bookbinder:
 
-Bookbinder is meant to be used from within a project called a **book**. The book includes a configuration file that describes which documentation repositories to use as source materials. Bookbinder provides a set of scripts to aggregate those repositories and publish them to various locations.
+1. Add gem "bookbindery" to your Gemfile.
```

Comment:
@upthewaterspout I think you're right: The Gemfile checked into the geode-book directory should already have bookbindery and shouldn't need to be added again.